query-algebrizer/src/cc.rs

```diff
@@ -334,10 +334,25 @@ impl ConjoiningClauses {
         self.value_bindings.get(var).cloned()
     }
 
-    pub fn bind_column_to_var(&mut self, table: TableAlias, column: DatomsColumn, var: Variable) {
+    pub fn bind_column_to_var(&mut self, schema: &Schema, table: TableAlias, column: DatomsColumn, var: Variable) {
```

Comment:
It's awkward to thread the schema-for-interpretation around piece-meal in this way.  Consider either pushing the `&Schema` reference into `CC`, or adding a `CCBuilder` to keep it around.  This could be a future clean-up/unification if you so choose.